### DISCUSSION TITLE
Implementing Nested Dropdowns with the Mantine Combobox that can be navigated through using up/down/left/right arrows

### DISCUSSION THREAD

**Discussion by Chiazokam** [mantinedev/mantine#6533] [Q&A]
Hi,

I am trying to implement a  dropdown with a submenu (a nested dropdown) using Mantine's Combobox component `(7.11.2)`. Going through the examples in the docs, I don't think there is an application to using the component to show submenus like so:

<img width="537" alt="image" src="https://github.com/user-attachments/assets/0eaf3cda-ed16-4601-9fc0-c8bd2cf3314a">

Please is there a work around for this in v7.11? Or is there a plan to support it?

Just like one can use up/down arrow keys to navigate through the dropdown, I also want to be able to use left/right arrow keys to navigate in/out of the submenu.

**Comment by rtivital** [comment]
Currently, nested menus are not supported. This feature is in the backlog, but it is not planned to be added soon as it might require breaking changes to the Menu component.

**Comment by Chiazokam** [comment]
Oh, thank you @rtivital
